tests/e2e/adu/service/common/e2e_test_core.ts

```diff
@@ -0,0 +1,660 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+

```

Comment:
Does the style checker work for files? If it's not something that makes sense to add to the repo, it could be good to quickly run the new files through something to make the semi-colon usage more consistent